tests/phpunit/tests/post/wpUpdatePost.php

```diff
@@ -0,0 +1,237 @@
+<?php
+/**
+ * Test wp_update_post() function
+ *
+ * @package WordPress
+ * @subpackage Post
+ *
+ * @since 6.9.0
+ */
+
+/**
+ * Class to Test wp_update_post() function
+ *
+ * @group post
+ * @covers ::wp_update_post
+ */
+class Tests_Post_WpUpdatePost extends WP_UnitTestCase {
+
+	/**
+	 * User IDs for the test.
+	 *
+	 * @var array{administrator: null, editor: null, contributor: null}
+	 */
+	protected static $user_ids = array(
+		'administrator' => null,
+		'editor'        => null,
+		'contributor'   => null,
+	);
+
+	/**
+	 * Set up before class.
+	 *
+	 * @param WP_UnitTest_Factory $factory The Unit Test Factory.
+	 */
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		self::$user_ids = array(
+			'administrator' => $factory->user->create(
+				array(
+					'role' => 'administrator',
+				)
+			),
+			'editor'        => $factory->user->create(
+				array(
+					'role' => 'editor',
+				)
+			),
+			'contributor'   => $factory->user->create(
+				array(
+					'role' => 'contributor',
+				)
+			),
+		);
+
+		$role = get_role( 'administrator' );
+		$role->add_cap( 'publish_mapped_meta_caps' );
+		$role->add_cap( 'publish_unmapped_meta_caps' );
+	}
+
+	/**
+	 * Tear down after class.
+	 */
+	public static function tear_down_after_class() {
```

Comment:
```suggestion
	public static function wpTearDownAfterClass() {
```

Reply:
See https://github.com/WordPress/wordpress-develop/blob/6ae67d9bf64df5cea4be9468163c63f344e874fa/tests/phpunit/includes/abstract-testcase.php#L87-L89

It behaves the same way as `wpSetUpBeforeClass()` in that it doesn't require a call to the parent. If you search the code base the use of `wpTearDownAfterClass` is more common that.